MdeModulePkg/Bus/Pci/PciBusDxe/PciEnumerator.c

```diff
@@ -300,10 +300,16 @@ ProcessOptionRom (
     }
 
     if ((Temp->RomSize != 0) && (Temp->RomSize <= MaxLength)) {
-      //
-      // Load and process the option rom
-      //
-      LoadOpRomImage (Temp, RomBase);
+      if (!Temp->IgnoreROM) {
+        // MU_CHANGE Start
+        DEBUG ((DEBUG_INFO, "Loading option rom from device at BDF=%d/%d/%d\n", Temp->BusNumber, Temp->DeviceNumber, Temp->FunctionNumber));

```

Comment:
Nit: These should technically be `%u` specifiers since the integers are unsigned.

```suggestion
        DEBUG ((DEBUG_INFO, "Loading option rom from device at BDF=%u/%u/%u\n", Temp->BusNumber, Temp->DeviceNumber, Temp->FunctionNumber));
```